src/main/java/com/ait/lienzo/shared/core/types/Color.java

```diff
@@ -669,7 +687,7 @@ private static double hueToRGB(double m1, double m2, double h)
     @Override
     public boolean equals(Object other)
     {
-        if ((other == null) || (false == (other instanceof Color)))
+        if (false == other instanceof Color)
```

Comment:
null never will be "instanceof Something" resp. java specification. Tests present.